solarforecastarbiter/reports/main.py

```diff
@@ -155,6 +156,36 @@ def get_validation_issues():
     return test
 
 
+def validate_resample_align(report, metadata, data):
```

Comment:
This should have it's own test in ``test_reports_main.py``

Reply:
Can do...